<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Download Manager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTreeWidget" name="tree">
      <attribute name="headerMinimumSectionSize">
       <number>0</number>
      </attribute>
      <column>
       <property name="text">
        <string>Id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Extensions</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>490</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionNewFilter"/>
    <addaction name="actionEditFilter"/>
    <addaction name="actionDeleteFilter"/>
    <addaction name="separator"/>
    <addaction name="actionSetFolder"/>
    <addaction name="actionStartProgram"/>
    <addaction name="actionStopProgram"/>
    <addaction name="separator"/>
    <addaction name="actionDestroy"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="actionNewFilter">
   <property name="text">
    <string>New filter</string>
   </property>
  </action>
  <action name="actionEdit_selected_filter">
   <property name="text">
    <string>Edit selected filter</string>
   </property>
  </action>
  <action name="actionEditFilter">
   <property name="text">
    <string>Edit selected filter</string>
   </property>
  </action>
  <action name="actionStartProgram">
   <property name="text">
    <string>Start program</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionDeleteFilter">
   <property name="text">
    <string>Delete filter</string>
   </property>
  </action>
  <action name="actionSetFolder">
   <property name="text">
    <string>Set downloads folder</string>
   </property>
  </action>
  <action name="actionStopProgram">
   <property name="text">
    <string>Stop program</string>
   </property>
  </action>
  <action name="actionDestroy">
   <property name="text">
    <string>Destroy</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
